vpdq/python/tests/test_util.py

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
 import vpdq
 import typing as t
+from pathlib import Path
+from typing import List, Union
```

Comment:
(ignorable, YMMV): Newer versions of python allow you to use `list[]` in places where we previously used List. I often find myself using `import typing as t` in older version because your list of imported types usually grows to need many lines

Reply:
Yes they do allow list, but this commit actually reverts this because I'm developing on Python 3.11 and used list[] which has been allowed since 3.9 but the CI runs a test on 3.8.